src/OVGA.cpp

```diff
@@ -926,28 +926,28 @@ static void init_dpi()
    shcoreDLL = SDL_LoadObject("SHCORE.DLL");
    if (shcoreDLL)
    {
-      SetProcessDpiAwareness = (HRESULT(WINAPI *)(PROCESS_DPI_AWARENESS)) SDL_LoadFunction(shcoreDLL, "SetProcessDpiAwareness");
```

Comment:
Textual find&replace is too crude: don't replace the text within the SDL_LoadFunction string, else you'll never find the actual function.

Reply:
Whoops. Fixed!